panamaconverter/src/Convert.scala

```diff
@@ -22,9 +22,18 @@ object Convert extends Phase {
   def transform(annotations: AnnotationSeq): AnnotationSeq =
     annotations.flatMap {
       case c @ ChiselCircuitAnnotation(circuit) =>
-        Seq(c, PanamaCIRCTConverterAnnotation(PanamaCIRCTConverter.convert(circuit, annotations.collectFirst {
-          case FirtoolOptionsAnnotation(firtoolOptions) => firtoolOptions
-        })))
+        Seq(
+          c,
+          PanamaCIRCTConverterAnnotation(
+            PanamaCIRCTConverter.convert(
+              circuit,
+              annotations.collectFirst {
+                case FirtoolOptionsAnnotation(firtoolOptions) => firtoolOptions
+              },
+              Some(firrtl.annotations.JsonProtocol.serialize(circuit.firrtlAnnotations.toSeq))
```

Comment:
Currently I don't have idea on how to filter these annos, we can use 
```
chisel3.panamalib.option.DisableUnknownAnnotations(true)
```
for unknown annotation behavior.